third_party/kubetest2/pkg/tester/e2e-tester.go

```diff
@@ -0,0 +1,142 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tester
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/octago/sflags/gen/gpflag"
+	"github.com/vladimirvivien/gexe"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/kubetest2/pkg/testers"
+)
+
+var GitTag string
```

Comment:
Might be a silly question. But where does this get substituted or updated ?

Reply:
I think the kubetest2 framework uses it. I copied from other example, it is worth exploring later whether it is needed or not.